catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceFilterDelegate.java

```diff
@@ -0,0 +1,243 @@
+
+    public static final Map<String, ConfluenceQueryParameter> QUERY_PARAMETERS;
+
+    private static final String PROPERTY_FORMAT = "%s %s \"%s\"";
+
+    private static final String DATE_FORMAT = "yyyy-MM-dd HH:mm";
+
+    private static final String WILD_CARD = "*";
+
+    private boolean wildcardQuery = false;
+
+    private int parameterCount = 0;
+
+    private Set<String> queryTags = new HashSet<>();
+
+    static {
+        Map<String, ConfluenceQueryParameter> params = new HashMap<>();
+        params.put(Metacard.ID,
+                new ConfluenceQueryParameter("id", false, true, false, false, true));
+        params.put(Metacard.CONTENT_TYPE,
+                new ConfluenceQueryParameter("type", false, true, false, false, true));
+        params.put(Metacard.TITLE,
+                new ConfluenceQueryParameter("title", true, true, false, true, false));
+        params.put(Metacard.ANY_TEXT,
+                new ConfluenceQueryParameter("text", true, false, false, true, false));
+        params.put(Metacard.CREATED,
+                new ConfluenceQueryParameter("created", false, true, true, false, false));
+        params.put(Metacard.MODIFIED,
+                new ConfluenceQueryParameter("lastmodified", false, true, true, false, false));
+        params.put(Topic.CATEGORY,
+                new ConfluenceQueryParameter("type", false, true, false, false, true));
+        params.put(Contact.CREATOR_NAME,
+                new ConfluenceQueryParameter("creator", false, true, false, true, true));
+        params.put(Contact.CONTRIBUTOR_NAME,
+                new ConfluenceQueryParameter("contributor", false, true, false, true, true));
+        params.put(Topic.KEYWORD,
+                new ConfluenceQueryParameter("label", false, true, false, false, true));
+        params.put(Core.METACARD_CREATED,
+                new ConfluenceQueryParameter("created", false, true, true, false, false));
+        params.put(Core.METACARD_MODIFIED,
+                new ConfluenceQueryParameter("lastmodified", false, true, true, false, false));
+        QUERY_PARAMETERS = Collections.unmodifiableMap(params);
+
+    }
+
+    @Override
+    public String and(List<String> operands) {
+        return operands.stream()
+                .filter(Objects::nonNull)
+                .collect(Collectors.joining(" AND ", "( ", " )"))
+                .replaceAll("\\(\\s*\\)", "");
+    }
+
+    @Override
+    public String or(List<String> operands) {
+        return operands.stream()
+                .filter(Objects::nonNull)
+                .collect(Collectors.joining(" OR ", "( ", " )"))
+                .replaceAll("\\(\\s*\\)", "");
+    }
+
+    @Override
+    public String not(String operand) {
+        return "NOT " + operand;
+    }
+
+    @Override
+    public String propertyIsEqualTo(String propertyName, String literal, boolean isCaseSensitive) {
+        ConfluenceQueryParameter param = QUERY_PARAMETERS.get(propertyName);
```

Comment:
Looks like the following 5 lines are duplicated 5 times.  Might be worth extracting a private function for these checks.